lib/ExpensiMark.js

```diff
@@ -654,9 +656,6 @@ export default class ExpensiMark {
             if (textToFormat !== '') {
                 replacedText += this.formatTextForQuote(regex, textToFormat, replacement);
             }
-
-            // Replace all the blank lines between quotes, if there are only blank lines between them
-            replacedText = replacedText.replace(/(<\/blockquote>((\s*)+)<blockquote>)/g, '</blockquote><blockquote>');
```

Comment:
It's safe to remove this as it's not covered by any unit test. 
I think we should retain line breaks between quotes because the App prefers to display user's input as it is.

Reply:
hmmm can you look into when it was added and why please?

Reply:
@iwiznia The change was added in this PR https://github.com/Expensify/expensify-common/pull/385 which is quite old. From the corresponding original issue https://github.com/Expensify/App/issues/2670, I found that it fixed retaining only one empty line within a quote if there’re more than one. But later we wanted to retain all empty rows inside a quote, see slack discussion https://expensify.slack.com/archives/C01GTK53T8Q/p1682331970891359.

This change was included at that time but no unit test case was added together in PR https://github.com/Expensify/expensify-common/pull/385. I have fixed another issue https://github.com/Expensify/expensify-common/pull/531 to avoid removing line break before and after heading. The App has been reported many issues about line break handling. And I found that we prefer to _**keep line breaks from user's input as it is instead of removing them**_, which is different from markdown rendering on Github. 

So, I think, in this case, we can also remove it to keep line breaks between quotes. Strictly speaking, it can be out of the scope of this issue but given it's annoying when performing tests and is easy to fix without breaking existing tests, I prefer to fix it and add tests to cover it together in this PR.